adafruit_monsterm4sk.py

```diff
@@ -62,7 +68,7 @@ class MonsterM4sk:
     The right screen is the one USB port directly above it.
     """
 
-    def __init__(self, i2c=None):
+    def __init__(self, i2c: Optional[I2CDeviceDriver] = None):
```

Comment:
I think `i2c` can be the `I2C` from busio like this:

```
from busio import I2C
# ....
def __init__(self, i2c: Optional[I2C] = None):
```

Reply:
@FoamyGuy Thanks for the tip, changes have been made.